analysis_options.yaml

```diff
@@ -97,6 +97,7 @@ linter:
 #    - prefer_interpolation_to_compose_strings
     - prefer_is_empty
     - prefer_is_not_empty
+    - prefer_relative_imports
```

Comment:
I don't think we want to enable this here since it will break all over the place.  (We tend to use absolutes in the linter code base.)